src/tasks/04_calculateSum.js

```diff
@@ -13,3 +13,13 @@
  */
 
 // <-- тут реализация
+function calculateSum() {
+	let numberArray = [];
+	for (let i = 0; i < arguments.length; i++) {
+		numberArray.push(arguments[i]);
+	}
```

Comment:
При неизвестном заранее количестве аргументов можно пользовать оператором "rest", он "собирает" все аргументы в массив (как раз что ты и делаешь в for цикле)
```suggestion
function calculateSum(...numberArray) {
```